app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -1214,7 +1276,10 @@ private void assertCorrectDays(){
     private void logInTestUser() {
         _solo.enterText(_solo.getEditText("Email"), _testUser.getEmail());
         _solo.enterText(_solo.getEditText("Password"), _testUser.getPassword());
+       //  _solo.enterText(_solo.getEditText("Email"), "julie@gmail.com");
+        // _solo.enterText(_solo.getEditText("Password"), "password");
 
         _solo.clickOnButton(LOGIN_TEXT);
+        _solo.sleep(4000);
```

Comment:
I think @jaskim9824's PR adding the wait already went in, so we probably don't need this anymore as this would be 8 total seconds of just idling in **all of our tests**.